packages/zoe/src/zoeService/utils.d.ts

```diff
@@ -65,6 +65,12 @@ type StartContractInstance<C> = (
   adminFacet: AdminFacet;
 }>;
 
+/** The result of `startInstance` */
+export type StartedInstanceKit<SF> = {
+  instance: Instance;
```

Comment:
Have you looked into parameterizing the `Instance` type such that `E(zoe).getPublicFacet(instance)` would get a useful type?

Reply:
I had but ran into some problem I don't remember. Thanks for the prompt; I got it into this PR.  PTAAL.

Reply:
@dckc I had expected Github to not allow merge without your second review because I clicked re-request review and that cleared the checkmark. 🤷